dnf/automatic/emitter.py

```diff
@@ -48,6 +50,7 @@ def _prepare_msg(self):
         if self._applied:
             msg.append(APPLIED % self._system_name)
             msg.append(self._available_msg)
+            msg.append(APPLIED_TIMESTAMP % time.ctime())
```

Comment:
The `time.ctime()` method doesn't print the date in the system's locale, you can use `time.strftime("%c")` to do that.

Reply:
That's much better.  I made that change.  Thank you.